internal/service/dns_config/model_config_display_view.go

```diff
@@ -0,0 +1,87 @@
+package dns_config
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/dns_config"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type ConfigDisplayViewModel struct {
```

Comment:
Is this model used anywhere? If not, it can be deleted.

Reply:
This is being used in model_config_server.go